packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Note: test coverage for this component exists in
+ * components/PaymentForm/index.test.tsx under the
+ * title 'rendering the legal checkbox Localized component'
```

Comment:
There are two places where we show this legal consent checkbox, one in `PaymentForm` and one in `SubscriptionUpgrade`. If we are going to make this consent checkbox a new component, we should probably dedupe the [consent checkbox in `SubscriptionUpgrade`](https://github.com/mozilla/fxa/blob/a63c7f05ef123bf964340f0a0b78cf41246421a2/packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/index.tsx#L195-L223) as well and move the `"rendering the legal checkbox Localized component"` tests for `PaymentForm` and `SubscriptionUpgrade` to a single test for the new component.